### PR TITLE
[SOC-5525]The document preview from an activity does not work if acti…

### PR DESCRIPTION
**Pull request by AtiaZ**
…vity message contains a link

Description : 
remove the extra HTML tags from the activityStatus.

### CODE REVIEW

**Comment by mtrabelsi** [comment]
LGTM

File: integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl [hunk 1]
```diff
@@ -177,7 +177,7 @@
   def docOpenUri = uicomponent.getCurrentDocOpenUri();
   def escapedOwnerName = ownerName.replace("'", "\\'").replace("&#39;", "\\&#39;");
   def escapedActivityPostedTime = activityPostedTime.replace("'", "\\'").replace("&#39;", "\\&#39;");
-  def escapedActivityStatus = activityStatus != null ? activityStatus.replace("'", "\\'").replace("&#39;", "\\&#39;") : "";
+  def escapedActivityStatus = activityStatus != null ? activityStatus.replace("'", "\\'").replace("&#39;", "\\&#39;").replaceAll("\\<.*?>","") : "";
```
Comment (thomasdelhomenie) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl]: Is this remove all the html tags ?

Comment (AtiaZ) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl]: Yes .. PR will be re-worked as discussed.

Comment (AtiaZ) [inline integ-ecms/integ-ecms-social/src/main/resources/groovy/ecm/social-integration/UISharedContent.gtmpl]: done

**Comment by AtiaZ** [comment]
Merged

**Pull request closed without merging**